discovery-infra/test_infra/assisted_service_api.py

```diff
@@ -13,19 +13,19 @@
 
 
 class InventoryClient(object):
-    def __init__(self, inventory_url):
+    def __init__(self, inventory_url, offline_token, pull_secret):
```

Comment:
we can have client without offline token and pull secret right? I think we must have defaults here

Reply:
they can be None though

Reply:
default is here:
 https://github.com/openshift/assisted-test-infra/blob/bd4328952346c3cdc17b2ca15f3b3bd3824fb35d/discovery-infra/test_infra/assisted_service_api.py#L323